sections/02_intro.md

```diff
@@ -38,16 +34,16 @@ inputs is usually multiplied by some weight and combined and summarized by the
 activation function. The value of the activation function is then multiplied by
 another set of weights to produce the output `TODO: we probably need a figure
 here - I see no way that we don't include this type of description in our paper,
-despite the fact that it's been done tons of times before.` These neural
+despite the fact that it's been done tons of times before. - I'm really partial to the nature bengio revew's presentation - really hitting on the idea of learning a feature to make a non-linear problem linear` These neural
 networks are trained by identifying weights that produce a desired output from
 some specific input.
 
 Neural networks can also be stacked. The outputs from one can be used as inputs
 to another. This process produces a stacked or multi-layer neural network. The
 multi-layer neural network techniques that underlie deep learning have a long
-history. Multi-layer methods have been discussed in the literature for more than
+history, having been discussed in the literature for more than
 five decades [@doi:10.1103/RevModPhys.34.135]. Given this context, it's
-challenging to consider "deep learning" as a new advance, though the term has
+challenging to consider "deep learning" as a new advance, `TODO: I'd argue the transition from multi-layer neural networks to deep learning is equivalent to the transition from theory to practice. The core concepts were there but architectures were discussed in the 80's, computationally we couldn't effectively train them until 00's this led to key breakthroughs in training, dropout, batch normalization, popularization of relu activation function. etc` though the term has
```

Comment:
Can you consider this change in the context of the most recent edits? It'd be nice to see a merge with these here. If you're citing specific techniques, you should reference the original papers on those advances to make it easy for a reader to follow.